docs/nlu/components.rst

```diff
@@ -256,23 +256,37 @@ CountVectorsFeaturizer
 Intent Classifiers
 ------------------
 
+
+.. _keyword_intent_classifier:
+
 KeywordIntentClassifier
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-:Short: Simple keyword matching intent classifier. Not intended to be used.
+:Short: Simple keyword matching intent classifier.
 :Outputs: ``intent``
 :Requires: nothing
```

Comment:
this is no longer true

Reply:
What do you actually mean by this? It requires training data only, that's not listed as required by any other classifiers.